src/Artsy/SystemContext.tsx

```diff
@@ -28,18 +36,20 @@ export interface SystemContextProps {
    */
   relayEnvironment?: Environment
 
+  /**
+   * An instance of react-tracking. When
+   */
+  tracking?: TrackingProp
+  initTracking?: (tracking: TrackingProp) => void
```

Comment:
A bit unsure of the naming here; this function is also _only_ called from within `useTracking()`, so it's effectively private.